tests/testthat/test-fims-estimation.R

```diff
@@ -633,8 +633,16 @@ test_that("run FIMS in a for loop", {
     catch <- em_input$L.obs$fleet1
     fishing_fleet_index <- new(fims$Index, length(catch))
     fishing_fleet_index$index_data <- catch
+    testindex <- 2
+    na_value <- -999
+    if(i==4){
+      fishing_fleet_index$index_data[testindex] <- na_value
```

Comment:
this is how R users would use it currently, correct? Assigning -999 to anything that should be an NA?

Maybe this should be added to the PR description at least for some minimal documentation on how this works.

Reply:
Good point - I added something to the fims-demo vignette about how to specify missing data!